CMake/resolve_dependency_modules/simdjson.cmake

```diff
@@ -34,3 +34,5 @@ if(${VELOX_SIMDJSON_SKIPUTF8VALIDATION})
 endif()
 
 FetchContent_MakeAvailable(simdjson)
+target_compile_definitions(simdjson
+                           PUBLIC SIMDJSON_EXPERIMENTAL_ALLOW_INCOMPLETE_JSON)
```

Comment:
Hi @leoluan2009, I am working on landing this but I am concerned  if this change also changes presto behavior ?
Are the changes affected by this flag only visible if iterate_allow_incomplete_json is called ?

Reply:
Looks like it only gates the allow_incomplete_json function : https://github.com/search?q=repo%3Asimdjson%2Fsimdjson%20SIMDJSON_EXPERIMENTAL_ALLOW_INCOMPLETE_JSON&type=code